mail_debrand/readme/HISTORY.rst

```diff
@@ -0,0 +1,14 @@
+[ The change log. The goal of this file is to help readers
```

Comment:
Remove this part

Reply:
is it not useful to future maintainers, just comments, I don't care.  Just seems strange to document the usage, then remove the documentation.

Reply:
Well, I care because it's just junk that doesn't serve. It's put on the template for knowing what to do on each file, but not for being preserved in all the new READMEs across whole OCA.